litestar/response/base.py

```diff
@@ -371,6 +372,9 @@ def render(self, content: Any, media_type: str, enc_hook: Serializer = default_s
         if isinstance(content, bytes):
             return content
 
+        if content is Empty:
+            raise RuntimeError("The `Empty` sentinel cannot be used as response content")
```

Comment:
we have a test that checks that when `Empty` is returned from a handler it raises a 500, so this preserves that behavior. Otherwise, as an enum `Empty` can be serialized.